nemo/collections/asr/parts/vad_utils.py

```diff
@@ -0,0 +1,272 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
The functions in this script are too generic. If you would like to keep the names, please import vad_utils such that all methods are called as `vad_utils.XYZ()`.